File: dev-docs/snippets/groups/update.swift
import JsBaoClient

// Update a group's name and/or description.
func update(client: JsBaoClient, groupType: String, groupId: String) async throws {
  // #region example
  let group = try await client.groups.update(
    groupType: groupType,
    groupId: groupId,
    params: ["name": "Design Guild", "description": "Renamed crew"]
  )
  // #endregion example
  _ = group
}

File: dev-docs/snippets/groups/update.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Update a group's name and/or description.
export async function update(
  client: JsBaoClient,
  groupType: string,
  groupId: string,
) {
  // #region example
  const group = await client.groups.update(groupType, groupId, {
    name: "Design Guild",
    description: "Renamed crew",
  });
  // #endregion example
  return group;
}
